addons/auth_signup/views/res_config_views.xml

```diff
@@ -1,9 +1,8 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<openerp>
-    <data>
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
 
         <record id="view_general_configuration" model="ir.ui.view">
-            <field name="name">base.config.settings.signup</field>
+            <field name="name">base.config.settings.form.inherit</field>
```

Comment:
don't change the name. that's making the inherit less clear. or if if you want to put "inherit" in it, keep what it does